libs/ui-components/src/icon/Icon.tsx

```diff
@@ -16,13 +16,16 @@ import { ReactComponent as CategoryMarineManager } from '../assets/icons/categor
 import { ReactComponent as CategoryNews } from '../assets/icons/category-news.svg'
 import { ReactComponent as CategoryReports } from '../assets/icons/category-reports.svg'
 import { ReactComponent as Close } from '../assets/icons/close.svg'
+import { ReactComponent as Clusters } from '../assets/icons/clusters.svg'
```

Comment:
IMHO this file deserves to be auto generated from an `ls assets/icons/`. Just an idea, maybe overkill.

Reply:
I like the idea! I would use something like [this](https://github.com/cycloidio/import-index-generator/blob/master/main.js#L44) as base to do it